pkgs/tools/admin/aws-vault/default.nix

```diff
@@ -32,6 +39,12 @@ buildGoModule rec {
     -X main.Version=v${version}
   '';
 
+  doInstallCheck = true;
+
+  installCheckPhase = ''
```

Comment:
I read something about this should now be in passthru tests but I am not sure.

Reply:
This is pretty far from anything we call test 🙂 Just a sanity check.

Where can I find more info on the agreements/conventions?

Reply:
@SuperSandro2000 this only applies to tests that take a long time to run. The idea is that for integration tests a second derivation is created that runs it, and doesn't block for the main derivation to build and produce an output. Here is should only add a few milliseconds so it's not worth the overhead of another derivation.

@ymatsiuk search for `passthru.tests` in the code base. It looks like most of them are using NixOS tests.

Reply:
Even though I see a use case for proper tests, this PR is not covering those. This is just a simple install check. I can also remove if you think this is not the best place for it.